rogueutil.h

```diff
@@ -76,6 +76,7 @@
 	#include <sys/ioctl.h> /* for getkey() */
 	#include <sys/types.h> /* for kbhit() */
 	#include <sys/time.h> /* for kbhit() */
+	#include <pwd.h> /* for getpwduid() */
```

Comment:
Looks like a typo, should be `getpwuid()`. Also, please add a comment on `getuid()` to the unistd.h include.

Reply:
Corrected.